test/integration/authorization_test.go

```diff
@@ -171,27 +171,27 @@ func TestClusterReaderCoverage(t *testing.T) {
 	// remove resources without read APIs
 	nonreadingResources := []schema.GroupResource{
 		buildapi.Resource("buildconfigs/instantiatebinary"),
-		buildapi.LegacyResource("buildconfigs/instantiatebinary"),
 		buildapi.Resource("buildconfigs/instantiate"),
-		buildapi.LegacyResource("buildconfigs/instantiate"),
 		buildapi.Resource("builds/clone"),
-		buildapi.LegacyResource("builds/clone"),
 		oappsapi.Resource("deploymentconfigrollbacks"),
-		oappsapi.LegacyResource("deploymentconfigrollbacks"),
 		oappsapi.Resource("generatedeploymentconfigs"),
-		oappsapi.LegacyResource("generatedeploymentconfigs"),
 		oappsapi.Resource("deploymentconfigs/rollback"),
-		oappsapi.LegacyResource("deploymentconfigs/rollback"),
 		oappsapi.Resource("deploymentconfigs/instantiate"),
-		oappsapi.LegacyResource("deploymentconfigs/instantiate"),
 		imageapi.Resource("imagestreamimports"),
-		imageapi.LegacyResource("imagestreamimports"),
 		imageapi.Resource("imagestreammappings"),
-		imageapi.LegacyResource("imagestreammappings"),
 		extensionsapi.Resource("deployments/rollback"),
 		appsapi.Resource("deployments/rollback"),
 		kapi.Resource("pods/attach"),
 		kapi.Resource("namespaces/finalize"),
+		{Group: "", Resource: "buildconfigs/instantiatebinary"},
```

Comment:
I think that authorization sticks around longer than we'll keep the admission endpoints.